.github/workflows/release_tools.yml

```diff
@@ -0,0 +1,87 @@
+name: "Create release of tools"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: "string"
+
+permissions:
+  id-token: write
+  attestations: write
+  contents: write
+
+jobs:
+  ci:
+    name: "CI"
+    uses: "./.github/workflows/ci.yml"
+  
+  create_tag:
+    name: "Create tag"
+    runs-on:
+      - "ubuntu-latest"
+    needs:
+      - "ci"
+
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: "Create tag"
+        id: "tag"
+        env:
+          VERSION: "${{ inputs.version }}"
+
+          GIT_AUTHOR_NAME: "${{ github.actor }}"
+          GIT_AUTHOR_EMAIL: "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
+          GIT_COMMITTER_NAME: "${{ github.actor }}"
+          GIT_COMMITTER_EMAIL: "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
+        run: |
+          # Update `MODULE.bazel` and commit.
+          bzlmod_names=("supply_chain_tools")
+          bzlmod_directories=("tools")
+
+          for bzlmod_directory in "${bzlmod_directories[@]}"; do
+            for bzlmod_name in "${bzlmod_names[@]}"; do
+              sed -i -e "s/version = .. /version = \"${VERSION}\" /" "${bzlmod_directory}/MODULE.bazel"
```

Comment:
This will also update `bazel_dep(..., version="1.2.3")`. Please change back to buildozer